espnet2/enh/layers/tcn.py

```diff
@@ -253,20 +258,37 @@ def forward(self, y):
         Returns:
             cLN_y: [M, N, K]
         """
+        dim = 3
+        if y.dim() == 4:
```

Comment:
Can you add a test for this case?

Reply:
This if-branch is never used in the codebase, so I removed this case.